docs/organizations/managing-security-and-risk.md

```diff
@@ -37,7 +39,7 @@ To access the item list, access the [dashboard](#dashboard) and click the **Revi
 
 When viewing the item list:
 
--   To update the filtering criteria, click the **Status** or **Severity** drop-downs above the list.
+-   To update the filtering criteria, click the **Severity**, **Status**, or **Repository** drop-downs above the list.
```

Comment:
```suggestion
-   To update the filtering criteria, click the **Severity**, **Status**, or **Repository** filters above the list.
```

Reply:
I see your point. In my view, we know they are filters, but they "look like" drop-downs.

Taking all the context out, I find "look for the drop-down" clearer than "look for the filter", so it may be cognitively easier (although yeah, it doesn't read as nice...)

Thoughts?

Reply:
I'm ok with dropdowns.

Reply:
ok, let's give it a go! :)